tests/eko/test_msbar_masses.py

```diff
@@ -1,128 +1,150 @@
 """
     Tests for the threshold class
 """
+from math import nan
+
 import numpy as np
 import pytest
 
 from eko import msbar_masses
-from eko.basis_rotation import quark_names
 from eko.couplings import Couplings
+from eko.io import types
+from eko.io.runcards import TheoryCard
+
 
-theory_dict = {
-    "alphas": 0.1180,
-    "alphaem": 0.00781,
-    "Qref": 91,
-    "nfref": 5,
-    "MaxNfPdf": 6,
-    "MaxNfAs": 6,
-    "Q0": 1,
-    "fact_to_ren_scale_ratio": 1.0,
-    "mc": 2.0,
-    "mb": 4.0,
-    "mt": 175.0,
-    "kcThr": 1.0,
-    "kbThr": 1.0,
-    "ktThr": 1.0,
-    "HQ": "MSBAR",
-    "Qmc": 2.1,
-    "Qmb": 4.1,
-    "Qmt": 174.9,
-    "order": (3, 0),
-    "ModEv": "TRN",
-    "ModSV": None,
-}
+@pytest.fixture()
+def theory_card(theory_card: TheoryCard):
```

Comment:
are you're sure you want to shadow here?

Reply:
Yes, this fixture is overwriting the other one for this module. Indeed it is just an update, and it depends from the broader.

Reply:
then you should explicitly silence pylint

Reply:
We are not pylinting tests:
https://github.com/NNPDF/eko/blob/e3466fae35fd245c7e1a99158d8582f590276555/pyproject.toml#L107-L108